theatre/studio/src/store/types/ahistoric.ts

```diff
@@ -1,8 +1,17 @@
 import type {ProjectState} from '@theatre/core/projects/store/storeTypes'
 import type {IRange, StrictRecord} from '@theatre/shared/utils/types'
+import type {Keyframe} from '@theatre/core/projects/store/types/SheetState_Historic'
+import type {PathToProp} from '@theatre/shared/utils/addresses'
+
+export type CopiedTrack = {
+  pathToProp: PathToProp
+  trackId: string
+  keyframes: Keyframe[]
+}
 
 export type StudioAhistoricState = {
   visibilityState: 'everythingIsHidden' | 'everythingIsVisible'
+  keyframesClipboard: CopiedTrack[]
```

Comment:
This is not backwards compatible. Someone using `theatre-0.4.0` for example, will not have the `keyframesClipboard` prop in their state.